mono/mini/mini-ops.h

```diff
@@ -1499,7 +1499,7 @@ MINI_OP(OP_MIPS_COND_EXC_INC, "mips_cond_exc_inc", NONE, IREG, IREG)
 
 #endif
 
-#if defined(TARGET_ARM64)
+#if defined(TARGET_ARM64) || defined(TARGET_AMD64)
```

Comment:
Same as above, these are arm specific instructions.

Reply:
so the reason for this is because i have to build the genmdesc project for windows x64 to build arm64 files which means the TARGET_ARM64 is not set